src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
```

Comment:
Use class names that represent the meaning of the content, not the styles or tag names. The class name '.body' can be misleading as it suggests styling for the 'body' tag, not a component or section of the page.